tools/retis_in_container.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -e
+set -ex
```

Comment:
guess, -x is a debugging leftover, right?

Reply:
hehe good catch, yes.